daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -320,7 +337,7 @@ func (d *Driver) subvolEnableQuota() error {
 func (d *Driver) subvolDisableQuota() error {
 	if !d.quotaEnabled {
```

Comment:
Shouldn't this be turned into a `isQuotaEnabled()` routine to avoid this code to be duplicated?

Also, `quotaEnabled` is just a boolean, but don't we need a lock around it?